docs/source/command_line.rst

```diff
@@ -448,9 +448,10 @@ potentially problematic or redundant in some way.
     are when:
 
     -   The function has a ``None`` or ``Any`` return type
-    -   The function has an empty body or a body that is just
-        ellipsis (``...``). Empty functions are often used for
-        abstract methods.
+    -   The function has an empty body and is marked as an abstractmethod,
+        is in a protocol class, or is in a stub file
+    -   For unreachable execution paths, for example, if an exception
+        is always raised
```

Comment:
```suggestion
    -  The execution path is unreachable; for example, if an exception
        is always raised
```
The list is introduced with "when", so each list item should complete the "when".